dwave/preprocessing/include/dwave-preprocessing/fix_variables.hpp

```diff
@@ -108,7 +109,8 @@ fixQuboVariables(dimod::AdjVectorBQM<V, B> &bqm, bool strict) {
   // Boros, Endre & Hammer, Peter & Tavares, Gabriel. (2006). Preprocessing of
   // unconstrained quadratic binary optimization. RUTCOR Research Report.
   double lower_bound = (posiform_info.getConstant() / posiform_info.getBiasConversionRatio())
```

Comment:
You can add this comment above. 

  // The bqm offset needs to be added to the lower bound because it was ignored
  //  while creating the posiform, this also reduces potential numerical errors.